pkg/hub/localizer/helm.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package localizer
+
+import (
+	"encoding/json"
+
+	"github.com/ghodss/yaml"
+	"helm.sh/helm/v3/pkg/chartutil"
+)
+
+// MergeChartValues merges src and override values into merged values for helm chart
```

Comment:
Could the input params just be `map[string]interface`?

For caller `aggregatedValues, err = MergeChartValues([]byte(override.OverrideValue["values"]), aggregatedValues)`, no need to have extra byte operations.